### PR TITLE
LPS-114698

### PR DESCRIPTION
**Pull request by brianchandotcom**

### CODE REVIEW

**Comment by brianchandotcom** [comment]
See https://github.com/brianchandotcom/liferay-portal/pull/90235

**Comment by brianchandotcom** [comment]
@brunobasto I need you to read my comments and apply them else where. Thanks! They are throw away commits.

**Comment by brunobasto** [comment]
Thank you @brianchandotcom.

File: modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/data/set/view/table/ClayTableSchema.java [hunk 1]
```diff
@@ -21,11 +21,11 @@
  */
 public class ClayTableSchema {
 
-	public Map<String, ClayTableSchemaField> getFields() {
+	public Map<String, ClayTableSchemaField> getClayTableSchemaFields() {
```
Comment (brunobasto) [inline modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/data/set/view/table/ClayTableSchema.java:24]: hey @brianchandotcom we are returning a Map here and not a ClayTableSchemaField. Do we still need to rename this?

Comment (brunobasto) [reply modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/data/set/view/table/ClayTableSchema.java:24]: Oh, looking more on the codebase I see the pattern.. Will resend

**Pull request closed without merging**